medmodels/medrecord/medrecord.py

```diff
@@ -787,21 +831,36 @@ def add_group(
 
         Args:
             group (Group): The name of the group to add.
-            node (Optional[Union[NodeIndex, NodeIndexInputList, NodeOperation]]):
+            nodes (Optional[Union[NodeIndex, NodeIndexInputList, NodeOperation]]):
+                Node index or indices to add to the group, optional.
```

Comment:
The description misses to explain how to use NodeOperation, and refers only to the indices.